### PR TITLE
[CDAP-17153] Add tabs for Preview tables when there are multiple stages

### PR DESCRIPTION
**Pull request by yukiej**
JIRA: https://issues.cask.co/browse/CDAP-17153
JIRA for tab styling: https://issues.cask.co/browse/CDAP-17171
Build: https://builds.cask.co/browse/CDAP-URUT312

Also made small changes to styling:
- for record view to not use Paper component, to be more consistent between table and record view (also the shadow from Paper was not necessary). 
- set height of horizontal tabs in ConfigurableTabs (instead of using fit-content) to be able to precisely align side by side tables if one side is using tabs.

### CODE REVIEW

File: cdap-ui/app/cdap/components/PreviewData/DataView/TableContainer.tsx [hunk 1]
```diff
@@ -116,29 +160,29 @@ const TableContainer: React.FC<IPreviewTableContainerProps> = ({
...
-                <DataTable
-                  headers={outputHeaders}
-                  records={outputRecords}
-                  isInput={false}
-                  previewStatus={previewStatus}
-                />
-              </div>
-            );
-          })}
+          {showOutputTabs
```
Comment (njbriggs) [inline cdap-ui/app/cdap/components/PreviewData/DataView/TableContainer.tsx]: Nit: Is there an easy way to consolidate this loop with the input table loop?

File: cdap-ui/app/cdap/components/Tabs/Tabs.scss [hunk 2]
```diff
@@ -41,7 +41,7 @@
       width: 100%;
       display: flex;
       &:first-child {
-        height: fit-content;
+        height: 42px;
```
Comment (njbriggs) [inline cdap-ui/app/cdap/components/Tabs/Tabs.scss]: Will this cause issues with other Tab instances?

**Review by njbriggs — COMMENTED** [review]
Mostly LGTM. 1 question and 1 possible point of regression.

File: cdap-ui/app/cdap/components/Tabs/Tabs.scss (hunk 2 above)
Comment (yukiej) [reply cdap-ui/app/cdap/components/Tabs/Tabs.scss]: The difference between fit-content and 42px is very small (fit-content was ~42.6px), and we only use horizontal tabs here and in one place in the schema editor. So if the fit-content change was OK for both places, this should be a minor change.

Comment (yukiej) [reply cdap-ui/app/cdap/components/Tabs/Tabs.scss]: @ajainarayanan Want to make sure you see this change to confirm that it is okay for the schema editor!

File: cdap-ui/app/cdap/components/PreviewData/DataView/TableContainer.tsx (hunk 1 above)
Comment (yukiej) [reply cdap-ui/app/cdap/components/PreviewData/DataView/TableContainer.tsx]: Good call! I've refactored to reduce the duplicate code here, and done the same for the RecordContainer as well.

**Review by njbriggs — APPROVED** [review]
LGTM!

File: cdap-ui/app/cdap/components/Tabs/Tabs.scss (hunk 2 above)
Comment (yukiej) [reply cdap-ui/app/cdap/components/Tabs/Tabs.scss]: Modified this slightly to still work if scrollbars are always set to on. Also tested in firefox and checked it worked ok in schema editor.

**Pull request merged**